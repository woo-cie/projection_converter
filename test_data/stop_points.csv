name,x,y,z,yaw,lane_id,stop_point_id,allow_goal_modification
StopPoint1,-0.4885,-1.0937,0.0000,1.5710,7,1,0
StopPoint2,-0.2129,9.6641,0.0000,1.5710,7,2,0
StopPoint3,-0.4885,23.2588,0.0000,1.5710,17,3,0
StopPoint4,4.1962,35.7513,0.0000,0.4580,130,4,0
StopPoint5,16.6887,37.2210,0.0000,0.0000,24,5,0
StopPoint6,-4.8057,36.3943,0.0000,2.6830,155,6,0
StopPoint7,-18.2167,37.6803,0.0000,3.1420,31,7,0
