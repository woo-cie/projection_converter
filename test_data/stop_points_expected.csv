name,x,y,z,yaw,lane_id,stop_point_id,allow_goal_modification
StopPoint1,-0.447652,-1.109614,0.0000,1.5710,7,1,0
StopPoint2,-0.565797,9.639144,0.0000,1.5710,7,2,0
StopPoint3,-1.337868,23.199612,0.0000,1.5710,17,3,0
StopPoint4,2.881837,35.841165,0.0000,0.4580,130,4,0
StopPoint5,15.298413,37.764925,0.0000,0.0000,24,5,0
StopPoint6,-6.127572,36.153952,0.0000,2.6830,155,6,0
StopPoint7,-19.561750,36.947420,0.0000,3.1420,31,7,0
